server.js

```diff
@@ -5,17 +5,51 @@
  */
 
 const express = require('express')
+const request = require('request')
+const Config = require('./src/Config')
 const versoSpoof = require('./src/versoSpoof')
 
 const port = 8000
 const app = express()
 
-app.use(express.static(`${__dirname}/`))
+// Required for ElasticSearch proxy middleware to parse response body as JSON
```

Comment:
Note that `server.js` doesn't have test coverage. I'm not sure exactly how or what I'd test. I *could* consider extracting the middleware elsewhere so it's easier to test but that's not super clear to me at the moment.